packages/wallets/safe/src/index.ts

```diff
@@ -1,43 +1,45 @@
-import { WalletAdapterType } from '@reef-knot/types';
-import { SafeConnector } from 'wagmi/connectors/safe';
+import { safe } from 'wagmi/connectors';
+import type { WalletAdapterType } from '@reef-knot/types';
 
 export const id = 'safe';
 export const name = 'Safe';
 
-export const Safe: WalletAdapterType = ({ chains }) => {
-  const safeConnector = new SafeConnector({
-    chains,
-    options: {
-      allowedDomains: [/app.safe.global$/, /holesky-safe.protofire.io$/],
-      debug: false,
-    },
+const getSafeConnector = () =>
+  safe({
+    allowedDomains: [/gnosis-safe.io$/, /app.safe.global$/],
```

Comment:
Also, note that the actual values in the latest reef-knot are
`[/app.safe.global$/, /holesky-safe.protofire.io$/]`

Reply:
Good point, I can add an argument. Btw, how did it worked before? Since there was no additional arguments.
@alx-khramov, thank you for the latest value, I will update those.